base-href-rewrite-plugin error - Plugin "baseHrefRewrite" of type "undefined" is not found, can not store config

## 🐞 Bug report

### Description
Following the guide here: https://github.com/scullyio/scully/tree/main/libs/plugins/base-href-rewrite I'm trying to add the base-href-rewrite-plugin for all routes.  Running `npm run scully` fails with `Plugin "baseHrefRewrite" of type "undefined" is not found, can not store config`.


#### 🔬 Minimal Reproduction
<pre><code>
import { ScullyConfig, setPluginConfig } from '@scullyio/scully';
import { baseHrefRewrite } from '@scullyio/scully-plugin-base-href-rewrite';

const defaultPostRenderers = ['seoHrefOptimise', baseHrefRewrite];
setPluginConfig(baseHrefRewrite, { href: '/diehl-with-software/' });
export const config: ScullyConfig = {
  projectRoot: './apps/web/src',
  projectName: 'web',
  outDir: './dist/static',
  defaultPostRenderers,
  routes: {
    '/blog/:slug': {
      type: 'contentFolder',
      slug: {
        folder: './blog',
      },
    },
  },
};

</code></pre>

#### 💻Your Environment

**Angular Version:**

<pre><code>
@angular-devkit/architect         0.901.9
@angular-devkit/build-angular     0.901.0
@angular-devkit/build-optimizer   0.901.0
@angular-devkit/build-webpack     0.901.0
@angular-devkit/core              9.1.9
@angular-devkit/schematics        9.1.9
@angular/cdk                      9.2.4
@angular/cli                      9.1.0
@angular/material                 9.2.4
@ngtools/webpack                  9.1.0
@schematics/angular               9.1.9
@schematics/update                0.901.0
rxjs                              6.5.4
typescript                        3.8.3
webpack                           4.42.0
</code></pre>

**Scully Version:**

<pre><code>
"@scullyio/init": "0.0.28",
    "@scullyio/ng-lib": "0.0.27",
    "@scullyio/scully": "0.0.99",
</code></pre>

## 🔥 Exception or Error

<pre><code>
npm run scully

> @rpd10/diehl-with-software@0.0.0 scully /Users/rpd10/Projects/github.com/diehl-with-software
> scully

/Users/rpd10/Projects/github.com/diehl-with-software /Users/rpd10/Projects/github.com/diehl-with-software
0.0.93
Plugin "baseHrefRewrite" of type "undefined" is not found, can not store config
npm ERR! code ELIFECYCLE
npm ERR! errno 15
npm ERR! @rpd10/diehl-with-software@0.0.0 scully: `scully`
npm ERR! Exit status 15
</code></pre>

--- a/libs/plugins/base-href-rewrite/package.json
+++ b/libs/plugins/base-href-rewrite/package.json
@@ -8,7 +8,7 @@
     "url": "https://github.com/scullyio/scully/tree/main/libs/plugins/base-href-rewrite"
   },
   "license": "MIT",
-  "dependencies": {
-    "tslib": "^1.13.0"
+  "peerDependencies": {
+    "@scullyio/scully": "*"
   }
 }

--- a/libs/plugins/base-href-rewrite/package.json
+++ b/libs/plugins/base-href-rewrite/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@scullyio/scully-plugin-base-href-rewrite",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "main": "index.js",
   "author": "Sander Elias",
   "repository": {

--- a/libs/plugins/base-href-rewrite/src/lib/plugins-base-href-rewrite.ts
+++ b/libs/plugins/base-href-rewrite/src/lib/plugins-base-href-rewrite.ts
@@ -4,10 +4,10 @@ import {
   setMyConfig,
   getMyConfig,
   log,
-  yellow
+  yellow,
 } from '@scullyio/scully';
 
-export const baseHrefRewrite = 'baseHrefRewrite';
+export const baseHrefRewrite = Symbol('baseHrefRewrite');
 
 const baseHrefRewritePlugin = async (
   html: string,
@@ -38,7 +38,7 @@ const baseHrefRewritePlugin = async (
 };
 
 setMyConfig(baseHrefRewritePlugin, {
-  href: '/'
+  href: '/',
 });
 
-registerPlugin('render', 'baseHrefRewrite', baseHrefRewritePlugin);
+registerPlugin('render', baseHrefRewrite, baseHrefRewritePlugin);

